libr/anal/class.c

```diff
@@ -937,7 +937,7 @@ R_API RAnalClassErr r_anal_class_vtable_delete(RAnal *anal, const char *class_na
 // ---- PRINT ----
 
 
-static void r_anal_class_print(RAnal *anal, const char *class_name, bool lng) {
+R_API void r_anal_class_print(RAnal *anal, const char *class_name, bool lng) {
```

Comment:
find a better name for the `lng` parameter

Reply:
Maybe "detailed" or smth along these lines?